x-pack/docs/en/settings/monitoring-settings.asciidoc

```diff
@@ -70,6 +70,10 @@ You can update this setting through the
 
 Sets the timeout for collecting the cluster statistics. Defaults to `10s`.
 
+`xpack.monitoring.collection.node.stats.timeout: 60s`::
```

Comment:
Could you please just document the name of the setting here as this is more consistent with the rest of the docs?